### PR TITLE
Add private/create_withdrawal method for Account

### PR DESCRIPTION
**Pull request by samuel-tay**
Quantity left as str because there's no easy way to determine the precision of each Coin.

### CODE REVIEW

File: src/cryptocom/exchange/private.py [hunk 1]
```diff
@@ -314,3 +314,22 @@ async def listen_orders(self, pair: Pair) -> Order:
...
+            'currency': coin.name,
+            'amount': quantity,
+            'address': address
+        }
+        if client_wid:
+            data['client_wid'] = str(client_wid)
+        if address_tag:
+            data['address_tag'] = str(address_tag)
+        if network_id:
+            data['network_id'] = str(network_id)
```
Comment (mortyspace) [inline src/cryptocom/exchange/private.py:332]: should be enum

Comment (samuel-tay) [reply src/cryptocom/exchange/private.py:332]: Not too sure what you mean? Do you mean that network_ids should be classes?

Comment (mortyspace) [reply src/cryptocom/exchange/private.py:332]: If there are enum "Select the desired network, require the address to be whitelisted first. See default_network and network in get-currency-networks for the value." probably needed to add a generation struct like for pairs

Comment (mortyspace) [reply src/cryptocom/exchange/private.py:332]: will look also at CI something strange 60mi runs, should be timedout by 1-2 mins but they didn't(

**Comment by samuel-tay** [comment]
Finally had time to work on this. Added enum for networks and auto-generate from private/get-currency-networks

**Comment by mortydevcom** [comment]
@samuel-tay you can use working v2.1 version API

**Comment by samuel-tay** [comment]
@mortydevcom hey, thanks I have already updated my fork to work with the things I need, but I will test the latest update when I can.

This PR was to add the private/create_withdrawal method which is not in this repo.

**Pull request closed without merging**